plugin/build.gradle.kts

```diff
@@ -179,10 +146,8 @@ kotlin {
 tasks.withType<Test> {
     dependsOn("publishToMavenLocal")
     useJUnitPlatform()
-    maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
-    doFirst {
-        logger.lifecycle("maxParallelForks for '$path' is $maxParallelForks")
-    }
+    maxParallelForks = 6 // no point in this being higher than the number of gradle workers
```

Comment:
Can you explain this a bit more?

Reply:
when junit forks processes to run parallel tests, it dispatches each one to a gradle worker. On my machine, this calculation was resulting in about 11 forks, but since we only had 4 workers, only 4 would run at a time. i increased workers to 6, and then hard coded this to match. we may want to reduce the test forks even more if we want other gradle tasks to execute while tests are running